e2e/e2e.js

```diff
@@ -16,46 +16,46 @@ const path = require('path')
 require('dotenv').config({ path: path.join(__dirname, '.env') })
 
 let sdkClient = {}
-const tenantId = process.env['{{LIB_NAME}}_TENANT_ID']
+const orgId = process.env['{{LIB_NAME}}_TENANT_ID']
```

Comment:
Why is the env still called TENANT_ID ?

Reply:
Good point, I can change it to ORG_ID as well. The issue that was fixed here is: the internal field is now called sdkClient.orgId (see line 28: expect(sdkClient.orgId).toBe(orgId))

Reply:
@moritzraho : Added a commit where I renamed all occurrences from tenantId to orgId

Reply:
ok thanks